lib/src/Bootstrap.php

```diff
@@ -213,7 +213,7 @@ public function boot($options = array()) {
 
       $this->log->debug("Load settings file \"" . $settings . "\"");
       define('CIVICRM_SETTINGS_PATH', $settings);
-      $error = @include_once $settings;
+      $error = include_once $settings;
```

Comment:
For this one, my guess: inertia

It's a little hard to see now, but `Bootstrap.php` arose from reorganizing/consolidating various copies of `civicrm.config.php.XXX` -- and some copies [had suppression on this line](https://github.com/civicrm/civicrm-drupal/blob/7.x-4.3/civicrm.config.php.drupal#L106). My guess is that someone felt dissatisfied that `include_once` didn't provide enough recoverability, and no one had the mix of incentive/time/expertise to question it.

But I share your view that this line probably isn't doing much.

Reply:
#212